manifests/w/Waterfox/Waterfox/G6.0.2/Waterfox.Waterfox.installer.yaml

```diff
@@ -0,0 +1,44 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Waterfox.Waterfox
+PackageVersion: G6.0.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 7.0.0.0
```

Comment:
```suggestion
MinimumOSVersion: 7.0.0.0
ElevationRequirement: elevationProhibited
```

Reply:
But that's very wrong. The installer by default tries to elevate itself and if it does, it installs machine-wide; if it doesn't, it installs in user mode. Right clicking the installer and clicking "run as administrator" works on my machine and installing locally has no issue.

Reply:
@stephengillie

Reply:
Hi @aaronliu0130,

Validation runs by default in an elevated context, which led to the error here. But you seem to be stating the opposite should occur. How should we proceed?

Reply:
@stephengille The only options I see are 1. Remove user mode support (which was the case for previous updates) 2. Manual validation (of both install modes). Are there no other ways this could be fixed?

Reply:
Whoops, forgot to ping @stephengillie

Reply:
This is unexpected - both machine and user scope work, but without a scope it fails.